src/node/transaction.cpp

```diff
@@ -50,10 +51,10 @@ TransactionError BroadcastTransaction(NodeContext& node, const CTransactionRef t
     if (!node.mempool->exists(hashTx)) {
         // Transaction is not already in the mempool.
         TxValidationState state;
-        CAmount fee{0};
-        if (max_tx_fee) {
+        if (max_tx_fee > 0) {
             // First, call ATMP with test_accept and check the fee. If ATMP
             // fails here, return error immediately.
+            CAmount fee{0};
```

Comment:
I noticed that the declaration of `fee` here was moved to a closer scope. As `fee` does not get used outside of the scope this appears to not change behaviour.

Reply:
Yep, just reducing the scope :)